docs/_guide/publish.md

```diff
@@ -31,33 +31,52 @@ You should also create a README describing how to consume your component. A basi
 
 ## Transpiling with TypeScript
 
-To transpile element code from TypeScript to JavaScript, conigure tsconfig.json. Make sure you include the `"downlevelIteration": true` option.
+When compiling your code from TypeScript to JavaScript, we recommend targeting ES2017 with Node.js module resolution. See the examples below for suggested options in tsconfig.json.
 
-**tsconfig.json**
+### Targeting ES2017 (Recommended)
+
+The following JSON sample is a partial tsconfig.json that uses recommended options for targeting ES2017:
 
 ```json
-{
-  "include": ["src/*.ts"],
   "compilerOptions": {
-    "downlevelIteration": true,
-    "target": "es2017",
-    "module": "es2017",
+    "target": "ES2017",
+    "module": "ES2017",
     "moduleResolution": "node",
-    "lib": ["es2017"],
-    "experimentalDecorators": true
+    "lib": ["ES2017", "DOM"],
+    "experimentalDecorators": true,
+    "outDir": "path/to/your-output-dir"
```

Comment:
I think we could just leave this out altogether

Reply:
agreed